media-libs/libv4l/libv4l-1.28.1.ebuild

```diff
@@ -128,7 +128,7 @@ pkg_postinst() {
 	xdg_pkg_postinst
 	use utils && udev_reload
 
-	if use utils && [[ -n ${REPLACING_VERSIONS} ]] && ver_test 1.20.0 -ge ${REPLACING_VERSIONS%% *}; then
+	if use utils && ver_replacing -le 1.20.0; then
```

Comment:
Now I think about it, this should probably be `-lt`.

Reply:
Yes, I had wondered about it but then decided not to change the existing behaviour.

Updated.